rllib/evaluation/rollout_worker.py

```diff
@@ -843,7 +851,7 @@ def learn_on_batch(self, samples: SampleBatchType) -> Dict:
             builders = {}
             to_fetch = {}
             for pid, batch in samples.policy_batches.items():
-                if pid not in self.policies_to_train:
+                if not self.policies_to_train(pid, samples):
```

Comment:
we should call self.is_policies_to_train() or self._should_train_policy() right?
there are quite a few places below that also use self.policies_to_train(), which is a list now.

Reply:
Great catch, thanks!